main/src/ca/uwaterloo/flix/language/phase/Lowering.scala

```diff
@@ -123,6 +127,8 @@ object Lowering extends Phase[Root, Root] {
     lazy val SolveType: Type = Type.mkPureArrow(Datalog, Datalog)
     lazy val MergeType: Type = Type.mkPureUncurriedArrow(List(Datalog, Datalog), Datalog)
     lazy val FilterType: Type = Type.mkPureUncurriedArrow(List(PredSym, Datalog), Datalog)
+
+//    lazy val NewChannelType: Type = Type.mkImpureArrow()
   }
 
```

Comment:
The key in this phase will be to translate `GetChannel`, `PutChannel`, and `Select` into calls in `Channel.flix`.